.github/workflows/test.yml

```diff
@@ -23,10 +25,9 @@ jobs:
         run: |
             git config --global user.email "41898282+github-actions[bot]@users.noreply.github.com"
```

Comment:
is the mail `41898282+github-actions` spacial to us?
If its possible to change can we move it to environmental variables?